wadas/domain/database.py

```diff
@@ -823,14 +822,12 @@ def get_users(cls):
                 return session.execute(stmt)
             else:
                 return None
-        except Exception as e:
-            WADASErrorMessage(
-                "Failed to retrieve users data",
-                f"Failed to retrieve user data from db. "
-                f"Please make sure db is healthy and properly configured.\n\nError: {str(e)}",
-            ).exec()
+        except Exception:
+            logger.error("Please make sure db is healthy and properly configured.")
+            return None
         finally:
-            session.close()
+            if session:
+                session.close()
```

Comment:
The previous code was ok. Please revert it (in all places), as per my last comment.